package.json

```diff
@@ -3,12 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
```

Comment:
- [ ] [OPTIONAL] React produce a Warning, kindly fix this so your app is Error/Warning free

Reply:
@Ismailco Please, how do i fix this?